htdocs/includes/page_elems.php

```diff
@@ -2800,19 +2800,19 @@ public function getPatientUpdateForm($pid)
 					</td>
 				</tr>
 				<tr>
-					<td><u>Satellite Lab ID</u></td>
+					<td><u>Satellite Lab Name</u></td>
 					<td>
-						<select name='satellite_lab_id' id='satellite_lab_id' class='uniform_width'>
+						<select name='satellite_lab_name' id='satellite_lab_name' class='uniform_width'>
 							<option value=''>Select Satellite Lab</option>
```

Comment:
This isn't part of this PR, but this string should be localized. (doesn't have to be translated but please add it to the language files so it can be later.)